dist/datasource.js

```diff
@@ -159,9 +160,9 @@ function (angular, _, dateMath, moment) {
         //Round up to start of an interval
         //Width of bar chars in Grafana is determined by size of the smallest interval
         var roundedFrom = granularity === "all" ? from : roundUpStartTime(from, granularity);
-        if(dataSource.periodGranularity!=""){
+        if(timeZone!=""){
```

Comment:
are we removing option `periodGranularity` ? I thought we are going to add a new option called `respect dashboard timezone` and then use it accordingly